pytripgui/model/plot_model.py

```diff
@@ -23,18 +23,18 @@ def __init__(self):
         self.plane = "Transversal"
 
     def next_slice(self):
-        self.position = (self.position + 1) % self._max_position()
+        self.slice_number = (self.slice_number + 1) % self._max_position()
```

Comment:
Now it gets inconsistent, previously you've been using `position` and `max_position`, now you have `slice_nuimber` and `max_position`.
I'd suggest to use also `_max_slice_no` or something similar.

Reply:
Another idea for variable names:
1. `current_slice_no` instead of `slice_number`
2. `last_slice_no` instead of `_max_position`